src/pages/Home/index.js

```diff
@@ -0,0 +1,31 @@
+import styles from './Home.module.css';
+
+const HomePage = () => (
+  <div className={styles.container}>
+    <section>
+      <ul>
+        <li>
+          <span>Book 1</span>
+          <span>Author 1</span>
+        </li>
+        <li>
+          <span>Book 2</span>
+          <span>Author 2</span>
+        </li>
+        <li>
+          <span>Book 3</span>
+          <span>Author 3</span>
+        </li>
+      </ul>
+    </section>
+    <section>
+      <form>
+        <input type="text" name="book" placeholder="Book name" />
+        <input type="text" name="author" placeholder="Author name" />
+        <input type="submit" value="Add new Book" />
+      </form>
+    </section>
+  </div>
```

Comment:
- [ ] **[OPTIONAL]** You did a great job working on your Home page! 🚀 However, I would suggest that you extract at least one reusable component from this, this is because this markup can get messy very quickly, and extracting meaningful components can make reading your code as well as handling events easier! 🤓 🔧 

💡 _I can see three logical components here: Booklist, represented by the `<ul>`, a Book, represented by each `<li>`, and the NewBookForm, represented by the `<form>`!_